test.py

```diff
@@ -125,9 +142,16 @@ def test_compress_size(self):
         '''
         compressed_img = compress(img_path, 3)
         assert image_size(compressed_img) < 4/8 * image_size(img_path)
+        
         compressed_img = compress(img_path, 1)
         assert image_size(compressed_img) < 2/8 * image_size(img_path)
 
+        compressed_img = compress(img_path, 6)
+        assert image_size(compressed_img) < 7/8 * image_size(img_path)
+
+        compressed_img = compress(img_path, 8)
+        assert image_size(compressed_img) == image_size(img_path)
+
```

Comment:
I don't think we should add test cases with such large b values. The tests take a lot of time for compressions with b>5.

Reply:
I think for the tests it should be okay. The test images I created are very small. I just tried it and it's pretty fast with those images.

Reply:
Makes sense!